pkg/ksym/ksym_test.go

```diff
@@ -54,56 +59,108 @@ ffffffff8f6d1600 b xfrm_state_afinfo
 ffffffff8f6d1768 b xfrm_state_afinfo_lock
 ffffffff8f6d1770 b xfrm_state_gc_list
 ffffffff8f6d1780 b xfrm_napi_dev
+ffffffff8f6d15c4 b not_in_order
 		`),
 			}))
 
-	addr1 := uint64(0xffffffff8f6d14a4) + 1
-	addr2 := uint64(0xffffffff8f6d15e0) + 1
-	addr3 := uint64(0xffffffff8f6d1480) + 1
+	addr1 := uint64(0xffffffff8f6d14a4)
```

Comment:
Why did you remove +1 from here and add it to every call?
An explanation would be awesome. (also  as a comment)

Reply:
I found it confusing as the addresses themselves were at the offset zero of the functions we wanted to search for. I wanted to cover both intermediate ranges (such as `start_address` + `n` (where `n` is 1 in this case), as well as 'exact' matches when the requested address is the `start_address`.

Hope this makes sense. Will add some comments to the test cases